tutorials/Multimodal_VQA_Interpret.ipynb

```diff
@@ -134,7 +128,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "vqa_net = torch.nn.DataParallel(Net(num_tokens), device_ids=[0,1])\n",
+    "vqa_net = torch.nn.DataParallel(Net(num_tokens))\n",
```

Comment:
Removing fixed number of devices in order to be able to take advantage of all of them.